docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -84,6 +86,27 @@ gradlew eclipse
 . https://help.eclipse.org/2022-03/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm[Import the project to Eclipse]
 . Start creating your tests
 
+=== Visual Studio Code
+
+==== Prerequisites
+
+. Install https://code.visualstudio.com[Visual Studio Code].
+. Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studio[VIVIDUS Studio Extension].
+
+==== Open the project in Visual Studio Code
+
+. Open Visual Studio Code.
+. Click `File` -> `Open...`.
+. Open a folder with a VIVIDUS project.
+. Wait until the message `Project with the name '...' is loaded` appears.
+. Start creating your tests.
+
+==== Execute created tests
+
+. Open https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette[Command Palette] by clicking `View` -> `Command Palette...`.
```

Comment:
does it make sense to use https://docs.asciidoctor.org/asciidoc/latest/macros/ui-macros/#menu-macro-syntax ?